projects/plugins/crm/js/ZeroBSCRM.admin.invoicebuilder.js

```diff
@@ -1087,7 +1088,7 @@ function zbscrm_JS_output_tax_line( res, i, v ) {
 
 	var tax_id;
 
-	if ( typeof v.taxes === 'undefined' || v.taxes == null ) {
+	if ( typeof v.taxes === 'undefined' || v.taxes == null || v.taxes == '' ) {
```

Comment:
This is a great example of where `falsey` values are useful:

```suggestion
	if ( !v.taxes === 'undefined' ) {
```

Reply:
I'm very rusty with Javascript... but that reduction could work...

Reply:
Not a blocker. Just something to be aware of.